### ISSUE TITLE
Uploaded picture not showing in registration form and profile view.

### ISSUE THREAD

**Issue by curioswati-zz** [curioswati-zz/IIPS-Mobile-App#6]
While registering, when user selects a picture to upload, it is not shown in the register page at the time and also not shown in the profile view later. Might be some issue in fetching the picture's address from device memory.

**Comment by curioswati-zz** [comment]
The issue is bit different. The image is fetched properly. The problem is at the time of saving it in the application storage it gets saved blank that is why it does not show up in the DOM's img element.

**Comment by shubham1337** [comment]
So, is the image not uploading properly to the backend ?

**Comment by curioswati-zz** [comment]
yes, we can say that. When we select image from gallery we need to save it in the application's local storage. But while saving it gets saved blank.
